README.md

```diff
@@ -47,7 +47,7 @@ For example, you can specify `scala-kmeans` as the benchmark.
 The following is a complete list of command-line options.
 
 ```
-Renaissance Benchmark Suite 0.9.0
+Renaissance Benchmark Suite ?.?.?
```

Comment:
@vhotspur I have seen this locally too, could you investigate ?
In the meantime, @Fithos, you can just manually make sure that the version string is correct

Reply:
@farquet I did a clean clone and I cannot reproduce this. Perhaps you had running SBT daemon and have not reloaded the project after #114?

Reply:
There is an issue on JDK8 with the version read from the manifest. 
I opened an issue about the wrong version (issue #123)